src/rust-2018/128-bit-integers.md

```diff
@@ -0,0 +1,17 @@
+# 128 bit integers
+
+![Minimum Rust version: 1.26](https://img.shields.io/badge/Minimum%20Rust%20Version-1.26-brightgreen.svg)
+
+A very simple feature: Rust now has 128 bit integers!
+
+```rust
+let x: i128 = 0;
+let y: u128 = 0;
+```
+
+These are twice the size of `u64`, and so can hold more values. More specifically,
+
+* `u128`: `0` - `340,282,366,920,938,463,463,374,607,431,768,211,455`
```

Comment:
This kinda looks like an array; I'd use underscore instead as a separator.

Reply:
we used this text in the release blog post and people were not generally confused

Reply:
I suppose; however, `340_282_366_920_938_463_463_374_607_431_768_211_455` has the benefit of being legal Rust code :)